libs/dex/src/main/java/io/crate/data/BiArrayRow.java

```diff
@@ -29,22 +29,12 @@ public class BiArrayRow extends Row {
     private Object[] firstCells;
     private Object[] secondCells;
 
-    public BiArrayRow() {
-    }
 
     public BiArrayRow(Object[] firstCells, Object[] secondCells) {
         this.firstCells = firstCells;
         this.secondCells = secondCells;
     }
 
-    public void firstCells(Object[] cells) {
```

Comment:
Is this related ?

Reply:
no, just a cleanup. Can revert it or do it in a separate commit

Reply:
Pushed a dedicated commit